src/journeys/_shared/step-bootstrap-tenant.js

```diff
@@ -276,6 +280,23 @@ const stepsBootstrapExampleTenant = [
 										</>
 									),
 								},
+								{
+									value: 'conference',
+									render: (
+										<>
+											<Text>
+												Conference
+												<Newline />
+												Conference boilerplate.
+												<Newline />
+												<Text dimColor>
+													Example implementation:
+													https://conference-boilerplate.netlify.app/
```

Comment:
We probably need to switch this out with our standard demo domain e.g.: conference.superfast.shop